packages/sit-onyx/src/components/OnyxSelect/types.ts

```diff
@@ -84,6 +89,10 @@ export type OnyxSelectProps<TValue extends SelectOptionValue = SelectOptionValue
      * If you want to use a button instead, use the `optionsEnd` slot.
      */
     lazyLoading?: SelectLazyLoading;
+    /**
+     * How to truncate the option label / text content if it exceeds the max width.
+     */
+    truncation?: TruncationType;
```

Comment:
question: shouldn't we have the same pattern as in the checkboxGroup and let the truncation be declared on the options-level? BaseSelectOption is shared between checkboxGroup and select and already offers `truncation` 
```
export type SelectOption<TValue extends SelectOptionValue = SelectOptionValue> = Pick<
  BaseSelectOption<TValue>,
  "value" | "label" | "disabled" | "truncation"
> 
```

Reply:
In this case, what happens with the checkAll truncation?

Reply:
I think the check all will always be ellipsis but it's not expected to have such a long text that truncation gets relevant